yt/frontends/__init__.py

```diff
@@ -1,6 +1,7 @@
 __all__ = [
     "adaptahop",
     "ahf",
+    "amrex",
```

Comment:
I think we should also indicate that `"boxlib"` is deprecated. We don't have API for that but a comment would be better than nothing.

Reply:
How about `"boxlib", #the boxlib frontend is deprecated, use 'amrex'`?

Reply:
works for me